server/utils/jwtGenerator.js

```diff
@@ -0,0 +1,15 @@
+const jwt = require("jsonwebtoken");
+require("dotenv").config();
+
+function jwtGenerator(user_id) {
+  const payload = {
+    user: {
+      id: user_id
+    }
+  };
+  
+
+  return jwt.sign(payload, process.env.jwtSecret, { expiresIn: "1h" });
```

Comment:
This is cool, have you thought yet about how you will store the JWT on the client side?

Reply:
Thank you.Yes I have managed to  look through few tutorials and documentation,  hopefully it should work.